### PR TITLE
Chore/organization people UI changes

### PR DESCRIPTION
**Pull request by AVtheking**
**What kind of change does this PR introduce?**

refactoring color schemes for color blind people

**Issue Number:**

Fixes #
**Does this PR introduce a breaking change?**

NO

##Screenshots
image
image

## Release Notes

- **New Features**
	- Added new icons for enhanced visual representation in the `AddMember` and `OrganizationPeople` components.
	- Introduced the ability to remove members and admins with a new delete icon in the `OrganizationPeople` component.
	- New CSS classes added for improved styling consistency.

- **Bug Fixes**
	- Corrected color values and improved visual hierarchy across various components.

- **Style**
	- Updated background colors and text styles for buttons and dropdowns to enhance user experience.
	- Modified CSS variables to refine the overall color scheme of the application.

- **Chores**
	- Updated ESLint configurations and dependencies for better code quality and maintainability.

## Release Notes

- **New Features**
	- Enhanced color palette for improved visual consistency across the application.
	- New icons and button styles in the `AddMember` and `OrganizationPeople` components for better user interaction.
	- Introduction of new CSS classes for improved styling and accessibility.
	- Updated button styles and color feedback in the `LeftDrawerOrg` component for clearer navigation.

- **Bug Fixes**
	- Updated styles for various components to ensure consistent appearance.

- **Chores**
	- Improvements made to the GitHub Actions workflow for better branch checks and output visibility.
	- Enabled formatting checks in pre-commit hooks to ensure code quality.

### CODE REVIEW

**Comment by palisadoes** [comment]
Please fix the conflicting files

File: .husky/pre-commit [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # npm run format:fix
 # npm run lint:fix
 npm run lint-staged
-npm run typecheck
+# npm run typecheck
```
Comment (palisadoes) [inline .husky/pre-commit:7]: Why was this done? I

**Review by palisadoes — CHANGES_REQUESTED** [review]
Please explain the changes to the `pre-commit` file

File: .husky/pre-commit (hunk 1 above)
Comment (AVtheking) [reply .husky/pre-commit]: it was by mistake reverted it.

**Comment by palisadoes** [comment]
Please make sure coderabbit approves your changes

**Comment by palisadoes** [comment]
Can you start creating a single CSS file with this PR? Then you can update that file for all subsequent color scheme design changes.

It will help greatly in standardizing the look and feel

**Comment by palisadoes** [comment]
Please fix the failing tests

**Review by palisadoes — APPROVED** [review]

**Pull request merged**